src/containers/org/OrgContainer.js

```diff
@@ -2,124 +2,179 @@
+import type { RequestState } from 'redux-reqseq';
+
+import { DataSetSearchResultCard, OrgActionButton } from './components';
+
+import { BadgeCheckIcon } from '../../assets';
+import {
+  CrumbLink,
+  GapGrid,
+  SearchForm,
+  SpaceBetweenGrid,
+  Spinner,
+  StackGrid,
+} from '../../components';
+import { ATLAS_DATA_SET_IDS, ENTITY_SET_IDS, SEARCH } from '../../core/redux/constants';
+import {
+  selectOrganization,
+  selectSearchHits,
+  selectSearchPage,
+  selectSearchQuery,
+  selectSearchTotalHits,
+} from '../../core/redux/selectors';
 import { Routes } from '../../core/router';
-
-const { NEUTRAL } = Colors;
+import {
+  SEARCH_DATA,
+  SEARCH_ORGANIZATION_DATA_SETS,
+  clearSearchState,
+  searchOrganizationDataSets,
+} from '../../core/search/actions';
+import { MAX_HITS_10 } from '../../core/search/constants';
+
+const { PURPLE } = Colors;
 const { isNonEmptyString } = LangUtils;
-const { selectOrganization } = ReduxUtils;
-
-const Boxes = styled.div`
-  display: flex;
-  flex-wrap: wrap;
-  margin-top: 48px;
-
-  > div {
-    border: 1px solid ${NEUTRAL.N200};
-    display: flex;
-    flex-direction: column;
-    gap: 8px;
-    margin-bottom: 48px;
-    margin-right: 48px;
-    max-width: 335px;
-    padding: 24px 36px;
-    width: 100%;
-
-    > span {
-      color: ${NEUTRAL.N500};
-    }
-
-    > b {
-      font-size: 50px;
-    }
-  }
-`;
-
-const ManageLink = styled(CrumbLink)`
-  align-items: center;
-  font-size: 16px;
-
-  > svg {
-    font-size: 12px;
-    margin-left: 8px;
-    margin-top: 2px;
-  }
-`;
-
-type Props = {
+const {
+  isPending,
+  isStandby,
+  isSuccess,
+} = ReduxUtils;
+
+const OrgContainer = ({
+  organizationId,
+} :{|
   organizationId :UUID;
-};
+|}) => {
 
-const OrgContainer = ({ organizationId } :Props) => {
+  const dispatch = useDispatch();
 
-  const organization :?Organization = useSelector(selectOrganization(organizationId));
-  const atlasDataSetIds :Set<UUID> = useSelector(selectOrganizationAtlasDataSetIds(organizationId));
-  const entitySetIds :Set<UUID> = useSelector(selectOrganizationEntitySetIds(organizationId));
+  const searchOrgDataSetsRS :?RequestState = useRequestState([SEARCH, SEARCH_ORGANIZATION_DATA_SETS]);
 
-  const dataSetsPath = useMemo(() => (
-    Routes.ORG_DATA_SETS.replace(Routes.ORG_ID_PARAM, organizationId)
-  ), [organizationId]);
+  const organization :?Organization = useSelector(selectOrganization(organizationId));
+  const searchPage :number = useSelector(selectSearchPage(SEARCH_ORGANIZATION_DATA_SETS));
+  const searchQuery :string = useSelector(selectSearchQuery(SEARCH_ORGANIZATION_DATA_SETS));
+  const searchHits :Map = useSelector(selectSearchHits(SEARCH_ORGANIZATION_DATA_SETS));
+  const searchTotalHits :Map = useSelector(selectSearchTotalHits(SEARCH_ORGANIZATION_DATA_SETS));
+  const totalHits :number = searchTotalHits.get(ATLAS_DATA_SET_IDS) + searchTotalHits.get(ENTITY_SET_IDS);
```

Comment:
would it be better to pass defaults here?

Reply:
Defaults are set by `INITIAL_STATE_SEARCH` in `src/core/search/constants/index.js`.

Reply:
Also, searching data sets is completely different in another branch that I'm working on, just blocked by the backend.